src/Compute/Compute.Test/ScenarioTests/StrategiesVirtualMachineTests.ps1

```diff
@@ -105,7 +105,7 @@ function Test-SimpleNewVmWithUltraSSD
 		#As of now the ultrasd feature is only supported in east us 2 and in the size Standard_D2s_v3, on the features GA the restriction will be lifted
 		#Use the follwing command to figure out the one to use 
 		#Get-AzComputeResourceSku | where {$_.ResourceType -eq "disks" -and $_.Name -eq "UltraSSD_LRS" }
-        $x = New-AzVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel -Location "east US 2" -EnableUltraSSD -Zone 3 -Size "Standard_D2s_v3"
+        $x = New-AzVM -Name $vmname -Credential $cred -DomainNameLabel $domainNameLabel -Location "eastus2" -EnableUltraSSD -Zone 2 -Size "Standard_D2s_v3"
```

Comment:
Have we run this test against mooncake / government cloud?

Reply:
Are you talking specifically about the UltraSSD?

Reply:
Yes.